var/spack/repos/builtin/packages/mpark-variant/package.py

```diff
@@ -9,15 +9,16 @@
 class MparkVariant(CMakePackage):
     """C++17 `std::variant` for C++11/14/17"""
 
-    homepage = "https://mpark.github.io/variant"
-    url      = "https://github.com/mpark/variant/archive/v1.3.0.tar.gz"
+    homepage = "https://github.com/mpark/variant"
+    git     = "https://github.com/mpark/variant.git"
```

Comment:
Let's keep the URL in here nonetheless, since it will pick up the zip over cloning the whole repo:
```suggestion
    url      = "https://github.com/mpark/variant/archive/v1.4.0.tar.gz"
    git      = "https://github.com/mpark/variant.git"
```